src/OpenRiaServices.Client/Framework/Internal/MergeAttribute.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using System;
 
-namespace OpenRiaServices.Server.Data
+namespace OpenRiaServices.Client.Internal
 {
     [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false, Inherited = true)]
-    public class MergeAttribute : Attribute
+    class MergeAttribute : Attribute
```

Comment:
Should we mark it [Obsolete] as well ?

Reply:
I considered making the type public and obsolete, but then we need to add suppression of obsolete attribute in the code.

Since it is private and has almost no references (it is currently useless as private) I think it is better to just remove all related code in a separate PR.